deriva/transfer/download/processors/__init__.py

```diff
@@ -30,7 +32,11 @@
     "interpolation": InterpolationTransformProcessor,
     "strsub": StrSubTransformProcessor,
     "geo": GeoExportTransformProcessor,
-    "fasta": FastaExportTransformProcessor
+    "fasta": FastaExportTransformProcessor,
+    "prideMTD": PrideMTDExportTransformProcessor,
+    "prideCV": PrideCVExportTransformProcessor,
+    "prideFM": PrideFMExportTransformProcessor,
+    "prideSM": PrideSMExportTransformProcessor
```

Comment:
I would prefer a single class that can be parameterized on construction to operate in different modes rather than four separate classes with largely the same boilerplate code with only minor differences in output behavior.